apps/gro/controllers/confirm.js

```diff
@@ -27,32 +28,16 @@ ConfirmController.prototype.saveValues = function saveValues(req, res, callback)
 
     var subject = locali18n.translate('pages.email.information.subject');
 
-    var d = {
-      values: (function getSessionValues(data) {
-        var r = {};
-        for (var prop in data) {
-          r[prop] = data[prop];
-        }
-        return r;
-      }(req.sessionModel.attributes))
-    };
+    var d = {values: _.pick(req.sessionModel.attributes, _.identity)};
```

Comment:
@joefitter I think it's only done as more data is being added after such as the date and the email. I assume it's better to do it this way rather than adding the data straight to the sessionModel. What do you think?